.github/workflows/pr_push.yml

```diff
@@ -53,7 +55,7 @@ jobs:
     needs: [FastBuild]
     uses: ./.github/workflows/reusable_qemu.yml
     with:
-      short_run: true
+      short_run: ${{ github.event_name == 'schedule' && 'false' || 'true' }}
```

Comment:
Remove `ubuntu-22.04` build from Nightly as this seems to be doubled now.

Also, shouldn't this workflow be triggered as part of a Nightly build? Now this job's results will not be as easy to find as if they were run as part of a Nightly as there are plenty of pr_push runs and only one Nightly a day.

Reply:
1) I do not want to modify Nightly.
2) This workflow is not reusable, so AFAIK it cannot be run as a part of the Nightly build.

@lukaszstolarczuk what do you think?

Reply:
> I would ask a different question - did you observe a better code coverage with long QEMU run? TBH, I don't expect increase in number of covered lines.
> 

You are right :-) Closing ...